randnum.c

```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+
+int main() {
+	srand(time(0));
+	int r = rand();
+
+	int x;
+
+	while(r>10)
```

Comment:
This definitely gets the job done, but alternatively, you could have just written int r = rand() % 10 + 1; and that would have worked!